onboarding.md

```diff
@@ -141,17 +141,28 @@ Lab meeting consists of two components described below.
 - Journal Club
 - Braintrust
 
-**Journal Club:** We have a 15 minute journal club to start each lab meeting.
-For journal club, prepare a presentation of 4 papers.
-All except for one should have been published since your last journal club presentation.
-The content you discuss - specifically your summary of the papers - should be the product of thoughtful analysis.
-The presentation itself should be simple.
-During the discussion, please share why you picked the paper, its implication for your research, and any potential implications that touch on other research that is ongoing in the lab.
-For each paper, the presentation should consist of:
-
-1.  A title slide
-2.  An overview slide (usually a flow-chart of some sort from the paper, could also be an initial result that sets context).
-3.  The results figure that convinced you to pick this paper.
+**Journal Club:** This is an opportunity for students to strengthen their ability to comprehend and critically analyze research that has already been conducted.
+For journal club, everyone will be randomly split off into groups of two (three if an odd number of students in the lab).

```

Comment:
```suggestion
**Journal Club:** This is an opportunity for lab members to strengthen their ability to comprehend and critically analyze research that has already been conducted.
For journal club, everyone will be randomly split off into groups of two (or three if an odd number of lab members).

```
I guess we're all students in some sense, but this phrasing might be slightly more inclusive of postdocs, etc.